src/App.js

```diff
@@ -2,19 +2,22 @@ import { useState } from 'react';
 import { Routes, Route } from 'react-router-dom';
 import ExpensesPage from './pages/ExpensesPage.js';
 import AddExpensePage from './pages/AddExpensePage.js';
-import SampleData from './data/sampleData.js';
+import EditExpensePage from './pages/EditExpensePage.js';
 
 export default function App() {
   const [members, setMembers] = useState([]);
-  const [expenses, setExpenses] = useState(SampleData);
+  const [expenses, setExpenses] = useState([]);
+  const [editExpense, setEditExpense] = useState();
+
   return (
     <Routes>
       <Route
         path="/"
         element={
           <ExpensesPage
             expenses={expenses}
-            setExpenses={setExpenses}
+            setExpenses={value => setExpenses(value)}
+            setEditExpense={value => setEditExpense(value)}
```

Comment:
Usually when passing callback functions as props we use the prefix `on`.
```suggestion
            onSetExpenses={value => setExpenses(value)}
            onSetEditExpense={value => setEditExpense(value)}
```